src/views/Error/error.scss

```diff
@@ -0,0 +1,25 @@
+.error-default-message{
```

Comment:
The layout looks a little weird on my screen and it remains in the same position when I adjust the size of my browser window. I would advise against using `position: relative` in favor of the following:
```
.errorPage {
   display: flex;
   flex-direction: column;
   align-items: center;
}
```
and then remove the classes associated with the text and button elements.